docker-compose.yml

```diff
@@ -43,7 +43,7 @@ services:
     container_name: order-mongodb
     command: mongod --auth
     ports:
-      - 27017:27017
+      - 27018:27018
```

Comment:
I wonder whether it makes sense to specify it like this, since I thought that the services connect to 27017 anyways, but then again I do not know how this stuff works

Reply:
To be completely honest, I am also not sure how docker manages this internally. With the current set up, both db's seem to be used as expected.